src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let finalPrice = 0;
```

Comment:
Prefer using `const` over `let` for `finalPrice` since its value is not reassigned.